gcloud-java-core/src/main/java/com/google/gcloud/PageImpl.java

```diff
@@ -38,6 +40,37 @@
     Page<T> nextPage();
   }
 
+  /**
+   * Base class for page fetchers.
+   *
+   * @param <T> the value type that the page holds. {@code T} should implement
+   *     {@code java.io.Serializable}
+   * @param <O> the service options type
+   */
+  @SuppressWarnings("rawtypes")
```

Comment:
Oh, I guess that is needed because` O` extends the rawtype for `ServiceOptions`?